mkdocs.yml

```
@@ -8,59 +8,61 @@ repo_url: https://github.com/V1EngineeringInc/V1EngineeringInc-Docs/
 edit_uri: 'tree/master/docs/'
 
 nav:
- - Index: index.md
- - Editing mkdocs: mkdocs_info.md
+ - New Documents Home: index.md
 
- - MPCNC:
-   - Main: mpcnc/index.md
-   - Assembly:
-     - Table: mpcnc/table.md
-     - Conduit: mpcnc/conduit.md
-     - Corners, Rollers, Base: mpcnc/base.md
-     - Z-Axis: mpcnc/z-axis.md
-     - Middle Assembly: mpcnc/middle-assm.md
-     - Gantry: mpcnc/gantry.md
-     - Final: mpcnc/final.md
-     - Belts: mpcnc/belts.md
+ - Build:
+   - MPCNC: 
+     - Basics: mpcnc/index.md
+     - Instructions:

```

I suggest "Assembly". These are all instructions.
```suggestion
     - Assembly:
```